packages/odf/components/kms-config/kms-config.tsx

```diff
@@ -52,6 +27,34 @@ export const KMSConfigure: React.FC<KMSConfigureProps> = ({
 }) => {
   const { t } = useCustomTranslation();
 
+  const KMSProviders: KMSProvidersType = React.useMemo(
+    () => [
+      {
+        name: t('Vault'),
+        value: ProviderNames.VAULT,
+        Component: VaultConfigure,
+      },
+      {
+        name: t('Hyper Protect Crypto Services'),
+        value: ProviderNames.HPCS,
+        Component: HpcsConfigure,
+        allowedPlatforms: ['IBMCloud'],
+      },
+      {
+        name: t('Thales CipherTrust Manager (using KMIP)'),
+        value: ProviderNames.THALES,
+        Component: ThalesConfigure,
+      },
+      {
+        name: t('Azure Key Vault (Tech preview)'),
```

Comment:
plz revert it back like it was earlier... we do not translate product name...
```suggestion
        name: Vault,
        value: ProviderNames.VAULT,
        Component: VaultConfigure,
      },
      {
        name: Hyper Protect Crypto Services,
        value: ProviderNames.HPCS,
        Component: HpcsConfigure,
        allowedPlatforms: ['IBMCloud'],
      },
      {
        name: Thales CipherTrust Manager (using KMIP),
        value: ProviderNames.THALES,
        Component: ThalesConfigure,
      },
      {
        name: Azure Key Vault,
```